src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Windows.cs

```diff
@@ -371,30 +378,35 @@ void ClearContainerEventHandlers()
 		protected virtual void Dispose(bool disposing)
 		{
 			if (_isDisposed)
+			{
 				return;
+			}
 
 			_isDisposed = true;
 
 			if (!disposing)
+			{
 				return;
+			}
 
 			ClearContainerEventHandlers();
 
-			if (_element != null)
+			if (_element is View && ElementGestureRecognizers is not null)
```

Comment:
Could we change this line to be:
```suggestion
			if (_element is View && ElementGestureRecognizers is ObservableCollection<IGestureRecognizer> gestures)
```
And then use the local `gestures` field throughout.

Calling `ElementGestureRecognizers` multiple times, looks like it would do some work:

https://github.com/dotnet/maui/blob/ae20455793142ae0034bcfdf8f810ca66aec782a/src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Windows.cs#L69-L70